frontend/src/views/ContactUs.tsx

```diff
@@ -20,7 +20,7 @@ function ContactUsPage() {
           Email copied!&nbsp;&nbsp;✕
         </CopyIndicator>
       </CopyIndicatorContainer>
-      <LandingPageContainer>
+      <CircleBackgroundContainer>
```

Comment:
So this page doesn't have a circle background, so this name is a bit confusing (I thought you added that background to this page, though I think it looks better without for this page). Maybe rename that container or something?

Reply:
Haha yeah that can be a bit confusing - I renamed to DynamicWidthContainer - lmk if you have any alternative ideas for a name though